### ISSUE TITLE
ggdist gradientinterval() default is min, max NOT 90% CI

### ISSUE THREAD

**Issue by aaarchmiller** [USGS-VIZLAB/forecast-drb#8]
The default behavior for the `ggdist::gradientinterval()` is to plot minimum to maximum values, NOT the 90% confidence intervals. This sketched screenshot demonstrates that the gradient interval goes starts at the minimum value, not at the lower 90% CI value. 

<img width="642" alt="image" src="https://user-images.githubusercontent.com/19958841/173448401-6497c1aa-bd37-4380-9d18-2293557ad251.png">

The `ymin` and `ymax` values should be pulling from `quantile(probs = c(0.05, 0.95)` to represent the true 90% CI

**Comment by ceenell** [comment]
My understanding was that the `width` parameter was setting the CI in `stat_gradientinterval`

https://github.com/USGS-VIZLAB/forecast-drb/blob/b19bb38325133ae2fe721d9c178486676c906da8/3_visualize/src/plot_gradient.R#L43

**Comment by aaarchmiller** [comment]
I thought so too, but when I changed that, the only thing that changed was the actual slab width?

**Comment by aaarchmiller** [comment]
Yes, I think that's true that it applies to the `geom_interval` but not `geom_gradientinterval`. There *must* be a way to feed this `geom` with different intervals than min, max. I also changed the .width here:

with `.width = c(0)`

<img width="642" alt="image" src="https://user-images.githubusercontent.com/19958841/173450054-38a7e54b-c47d-4727-8e77-516aafb8ef99.png">


And here `.width = c(0.1)`

<img width="642" alt="image" src="https://user-images.githubusercontent.com/19958841/173450152-3465e210-2c88-4a2a-b3ec-1653af4c83f3.png">


I don't see a change

**Comment by aaarchmiller** [comment]
Another option is to just scrap this version for now, and move onto using the more flexible `geom_interval` where we can directly specify the CI level.

**Comment by ceenell** [comment]
OK form there I think we can either:
- pull the 90%CI from the [old data](https://code.usgs.gov/wma/wp/forecast-preprint-code/-/blob/main/in/all_mods_with_obs.rds) to restrict what is displayed
- add the black 90%CI bar using `.width`, or 
- accept that it shows the full range of predicted values. 

going with the last one for now

**Comment by aaarchmiller** [comment]
Another option besides the three listed above it so calculate the 90% min and max manually within the target with `quantile()` -- I used that function a lot with Bayesian posteriors. Then we wouldn't have to load the CIs from a different dataset.
